libxo.mk

```diff
@@ -0,0 +1,82 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += libxo
+LIBXO_VERSION := 1.4.0
+DEB_LIBXO_V   ?= $(LIBXO_VERSION)
+
+libxo-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) \
+		https://github.com/Juniper/libxo/releases/download/$(LIBXO_VERSION)/libxo-$(LIBXO_VERSION).tar.gz
+	$(call EXTRACT_TAR,libxo-$(LIBXO_VERSION).tar.gz,libxo-$(LIBXO_VERSION),libxo)
+
+ifneq ($(wildcard $(BUILD_WORK)/libxo/.build_complete),)
+libxo:
+	@echo "Using previously built libxo."
+else
+libxo: libxo-setup
+	cd $(BUILD_WORK)/libxo && ./configure -C \
+		$(DEFAULT_CONFIGURE_FLAGS) \
+		--with-gettext=$(BUILD_BASE)/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
+	$(SED) -i'' '/rpl_/d' $(BUILD_WORK)/libxo/libxo/xo_config.h
```

Comment:
Not immediately sure, but this can probably be fixed with a configure arg. ac_cv_has_rpl_malloc or something

Reply:
Also you should be using GNU sed here, why does it have the '' after the -I?